tests/foreman/longrun/test_inc_updates.py

```diff
@@ -0,0 +1,338 @@
+"""Tests for the Incremental Update feature"""
+
+from datetime import date, timedelta
+from fauxfactory import gen_alpha
+from nailgun import entity_mixins
+from nailgun.entities import (
```

Comment:
Lets tackle this here https://github.com/SatelliteQE/robottelo/issues/2760 instead of blocking this PR